crates/solver/src/liquidity/order_converter.rs

```diff
@@ -55,12 +54,7 @@ impl OrderConverter {
             available_sell_token_balance,
         )?;
 
-        let sell_amount = match &order.metadata.class {
-            OrderClass::Limit(limit) if !order.data.partially_fillable => {
-                compute_synthetic_order_amounts_for_limit_order(&order, limit)?
-            }
-            _ => order.data.sell_amount,
-        };
+        let sell_amount = order.data.sell_amount;
```

Comment:
or `let sell_amount = order.data.sell_amount + order.data.fee_amount`?

Reply:
```rust
let sell_amount = order.data.sell_amount;
```
looks correct to me. But for limit orders it shouldn't make a difference anyway since they have `fee_amount == 0`, right?

Reply:
I also think so, just wasn't sure why it was relevant up so far.

Reply:
You can just remove this line:
```suggestion
```

And replace 👇 `s/sell_amount/order.data.sell_amount/g`.

Reply:
`s/sell_amount/order.data.sell_amount/g` I don't get it?